.vsts-ci.yml

```diff
@@ -379,17 +372,6 @@ stages:
       vmrBranch: ${{ variables.VmrBranch }}
       isBuiltFromVmr: false
 
-# In case the VMR Build stage is temporarily disabled, the VMR synchronization step is run to validate
-# that the PR can be merged and later synchronized into the VMR without problems.
-- ${{ else }}:
-  - stage: Synchronize_VMR
-    displayName: Synchronize VMR
-    dependsOn: []
-    jobs:
-    - template: eng/pipelines/templates/jobs/vmr-synchronization.yml
-      parameters:
-        vmrBranch: ${{ variables.VmrBranch }}
-
```

Comment:
We should still run the sync in PRs in case the VMR build is disabled so that we have validation of the VMR sync running. So instead of `else` it should check if "VMR build is disabled and we're in PR"